en/08_Changelogs/6.0.0.md

```diff
@@ -898,6 +898,14 @@ The following classes have been changed when code was moved to the `silverstripe
 |`SilverStripe\ExternalLinks\Tasks\CurlLinkChecker`|[`SilverStripe\Reports\ExternalLinks\Tasks\CurlLinkChecker`](api:SilverStripe\Reports\ExternalLinks\Tasks\CurlLinkChecker)|
 |`SilverStripe\ExternalLinks\Tasks\LinkChecker`|[`SilverStripe\Reports\ExternalLinks\Tasks\LinkChecker`](api:SilverStripe\Reports\ExternalLinks\Tasks\LinkChecker)|
 
+The following subclasses of [`Validator`](api:SilverStripe\Forms\Validator) have been renamed:
+
+|Old FQCN|New FQCN|
+|---|---|
+|`SilverStripe\Forms\RequiredFields`|[`SilverStripe\Forms\RequiredFieldsValidator`](api:SilverStripe\Forms\RequiredFieldsValidator)|
```

Comment:
Need to update the namespace and add the `CompositeValidator`.

Reply:
Updated

Reply:
Still not done?? You added `CompositeValidator` but didn't update the namespaces of these two.

Reply:
Updated